packages/wonder-blocks-icon-button/src/themes/default.ts

```diff
@@ -37,7 +22,10 @@ const baseColorStates = {
             border: "transparent",
             background: "transparent",
         },
-        focus: focusOutline,
+        hover: {
```

Comment:
Question for later: Do we think global hover styles would be helpful too (similar to the global focus styles)? Or more on a component level basis?

From what I've noticed, we have a variety of hover styles: outline around the element, underline, left bar indicator. We could eventually have different hover styles in the styles package (ex: `hoverOutline`, `hoverUnderline`, etc). What do you think!